install/sql/update-2.5.2.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE glpi_plugin_resources_configs ADD `import_external_datas` tinyint(1) NOT NULL DEFAULT '0';
+ALTER TABLE glpi_plugin_resources_resources ADD `matricule_External` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL;
```

Comment:
The field name must be in lower case
use matricule__external
Idem for others fields